app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+    validates :title, presence: true
+    validates :content, presence: true
```

Comment:
in the db we are using title and body, and here we are using content, who come it is running ??

Reply:
Added missing file

Reply:
ok